nautobot/virtualization/tests/test_filters.py

```diff
@@ -415,28 +451,12 @@ def test_primary_ip6(self):
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
 
     def test_services(self):
-        with self.subTest("Services"):
-            params = {"services": [self.services[0].pk, self.services[1].name]}
-            self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
-
-        with self.subTest("Has Services"):
-            params = {"has_services": True}
-            self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
-
-            params = {"has_services": False}
-            self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
+        params = {"services": [self.services[0].pk, self.services[1].name]}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
```

Comment:
Can we switch here and below to using `assertQuerysetEqualAndNotEmpty` without too much additional work? Non-blocking.